server/router.ts

```diff
@@ -63,6 +65,10 @@ function createRouter(connection: Connection) {
             }).catch(next);
     });
 
+    router.get(nav.user.href, (req, res) => {
+        const id = req.body.id;
```

Comment:
You're gonna want to look at
https://expressjs.com/en/guide/routing.html#route-parameters
And
https://expressjs.com/en/4x/api.html#router.param

Things in `nav` are only for things which show up in the top navbar, and every use page won't show up there
Although you may want a link to a "Users" page for viewing all users